drizzle/0000_funny_paper_doll.sql

```diff
@@ -0,0 +1,535 @@
+-- Current sql file was generated after introspecting the database
```

Comment:
This is the original schema generated by Drizzle looking at the DB created from the Prisma schema